packages/suite/src/hooks/wallet/__fixtures__/useSendForm.ts

```diff
@@ -452,20 +452,20 @@ export const composeDebouncedTransaction = [
         actions: [
             {
                 type: 'input',
-                element: 'outputs[0].address',
+                element: 'outputs.0.address',
                 value: 'X',
                 result: {
                     errors: {
-                        outputs: [{ address: { message: 'RECIPIENT_IS_NOT_VALID' } }],
+                        outputs: [{ address: { type: 'valid' } }],
```

Comment:
@matejkriz since currently errors are stored already translated in the send form, what about using the english translation for the tests here until we further improve/unify validation with `yup` one day?

Reply:
I would prefer not to use english translation here, no reason to break the test if the translation changes...